### ISSUE TITLE
Add a ceasn:abilityEmbodied property

### ISSUE THREAD

**Issue by stuartasutton** [CredentialEngine/CompetencyFrameworks#76]
Currently, we have a [`ceasn:skillEmbodied`](http://purl.org/ctdlasn/terms/skillEmbodied) property with a domain of [`ceasn:Competency`](http://purl.org/ctdlasn/terms/Competency) that makes it possible for a specific competency to have a [`ceasn:competencyCategory`](http://purl.org/ctdlasn/terms/competencyCategory) of something like "Task" and then assert that that task has a `ceasn:skillEmbodied` of something like "Judgment and decision making." Current work on the Navy Task Classification indicates that we could benefit from a companion `ceasn:abilityEmbodied` property.

**Comment by siuc-nate** [comment]
Aren't both of those fundamentally covered by the notion of a Competency to begin with?  Would it also then follow that we need a `ceasn:knowledgeEmbodied`, `ceasn:behaviorEmbodied`, `ceasn:attitudeEmbodied`, and so on?

**Comment by stuartasutton** [comment]
No. Not where the skill or ability embodied are relevant to a specific competency node. We have no way to handle that with any precision. Plus, I see no reason to relitigate the existence of 'ceasn:skillEmbodied'. A lot of work went into that back when it was added to ASN. And, no, it does not follow that by adding a 'ceasn:skillEmbodied' we'd need to create an extended array of other properties absent some compelling need.

**Comment by Lomilar** [comment]
Skill embodied appears to be a method of tagging competencies with descriptive psycho and physical classifiers, like: Cognitive based skill, Psychomotor based skill, Empathy based skill, Visual skill, etc.

If that interpretation is correct, the word sense of 'skill' in skillEmbodied is different from 'skill' as denoted in KSAs (being the ability to recall knowledge, perform a skill, or exhibit an ability)

Let me know if I am off base. Was I starting from scratch, the ability to perform psychomotor type tasks would be a soft competency (like the ability to see or the ability to recall information) and there would be a requires relation established, but this isn't the worst way to do this. I'm not sure of all the implications either way.

**Comment by stuartasutton** [comment]
@Lomilar, not quite sure what you are driving at with the couple of sentences. If I understand you correctly, the only difference between what I (and the ASN) are saying is that you switch the `skillEmbodied` property for `requires`. The object is modeled as either a competency or concept scheme either way.

**Comment by Lomilar** [comment]
Correct, we don't yet perceive the nuance that distinguishes requiring a psychomotor ability from embodying a psychomotor skill, for instance.
